lib/plugin/swagger.js

```diff
@@ -42,6 +42,14 @@ const TeamRef = Joi.object({
 }).label('TeamRef')
 const TeamRefs = Joi.array().items(TeamRef).description('Team refs').label('TeamRefs')
 
+const ShortTeam = Joi.object({
+  id: Joi.string().description('Team ID'),
+  name: Joi.string().description('Team name'),
+  description: Joi.string().description('Team description'),
+  path: Joi.string(),
+  organizationId: Joi.string().description('Organization ID to which the team belongs to')
+}).label('Short Team')
```

Comment:
just one question here: do you think any of these should be required? we could enforce at joi level.
@cianfoley-nearform is this joi definition ok to be placed here?

Reply:
Hi Mihai, both are actually being used, I think there should be a common file with all base definitions e.g. policy, and components like resources, teams etc. and then reuse those in input and output validation... there might be a case for a specific http definition file, but it possibly shouldn't be called swagger.js, perhaps restValidation